kong/llm/schemas/init.lua

```diff
@@ -97,6 +97,11 @@ local auth_schema = {
         required = false,
         encrypted = true,
         referenceable = true }},
+    { can_override = {
```

Comment:
```suggestion
    { allow_auth_override = {
```

maybe this? To me, `can_override` is a little undescriptive.

Reply:
this is under the `auth` table, so `config.auth.can_override`. maybe `config.auth.allow_override`?

Reply:
`allow_override` seems reasonable here. I will change it